persistence-modules/hibernate5/src/main/java/com/baeldung/hibernate/onetoone/Strategy.java

```diff
@@ -2,6 +2,7 @@
 
 
 import java.util.Arrays;
+import java.util.Collections;
```

Comment:
Is this change necessary? It doesn't seem like the file changed otherwise? If not, please remove from the PR.